src/c_kzg_4844.h

```diff
@@ -32,6 +32,10 @@
 extern "C" {
 #endif
 
+#ifndef DLLEXPORT
+#define DLLEXPORT
+#endif
```

Comment:
Hm.. This feels slightly dirty to have on the core c_kzg_4844.h file. Let's at least document what this does.

Reply:
Fully agree, it's ugly but I've found no other way. A comment has been added

Reply:
If I initialize it in the common file, other bindings will have more exports than probably expected by them, not sure if it's OK.

Reply:
Ah okay. That's a good reason not to.